addon/runtime.js

```diff
@@ -34,6 +34,9 @@ export default function(addon) {
       // apply defaults
       optparse.mergeTrackerOpts(mapping, _settings);
       optparse.handlerOpts(mapping);
+      optparse.timingHandlerOpts(mapping);
```

Comment:
whereas `timingHandler` (defined in `configure`, `track` or provided by default) could be a part of particular timing `chain` before pushing into global `chains`.